lib/ansible/modules/windows/win_psrepository.ps1

```diff
@@ -1,68 +1,229 @@
 #!powershell
 
+# Copyright: (c) 2020, Brian Scholer <@briantist>
 # Copyright: (c) 2018, Wojciech Sciesinski <wojciech[at]sciesinski[dot]net>
 # Copyright: (c) 2017, Daniele Lazzari <lazzari@mailup.com>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 #Requires -Module Ansible.ModuleUtils.Legacy
+#Requires -Module @{ ModuleName = 'PowerShellGet' ; ModuleVersion = '1.6.0' }
```

Comment:
These statements don't do anything when run in PowerShell. If you wanted to provide a nice error in case the requirements isn't met then you need to do a manual check.